.github/workflows/README.md

```diff
@@ -698,7 +699,7 @@ jobs:
       pypi-token: ${{ secrets.TEST_PYPI_TOKEN }}
 ```
 
-## Java Gradle Docker
+## Java Gradle Base
 
 This workflow will build, test and publish a Java Gradle project including a tarball image. Additionally,
 the workflow creates a GitHub Release when running on a tag branch.
```

Comment:
This does not seem to be correct for `Java Gradle Base`, because no tarball image is being build and no GitHub release is created on tag branches

Reply:
oh yeah